sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -984,6 +993,20 @@ public partial class HttpPipelineTransportOptions
         public System.Collections.Generic.IList<System.Security.Cryptography.X509Certificates.X509Certificate2> ClientCertificates { get { throw null; } }
         public System.Func<Azure.Core.Pipeline.ServerCertificateCustomValidationArgs, bool>? ServerCertificateCustomValidationCallback { get { throw null; } set { } }
     }
+    public abstract partial class RetryPolicy : Azure.Core.Pipeline.HttpPipelinePolicy
+    {
+        protected RetryPolicy(Azure.Core.RetryOptions? options = null) { }
+        protected virtual System.TimeSpan CalculateNextDelay(Azure.Core.HttpMessage message) { throw null; }
```

Comment:
Any reason not to call this `ComputeDelay`?  It sounds a bit cleaner.

Reply:
I like having `Next` in there because it helps to clarify that this will be called for each retry attempt. I don't have a preference for Compute vs Calculate.